fileinfo.go

```diff
@@ -36,6 +38,8 @@ func SetFileBasicInfo(f *os.File, bi *FileBasicInfo) error {
 	if err := setFileInformationByHandle(syscall.Handle(f.Fd()), fileBasicInfo, (*byte)(unsafe.Pointer(bi)), uint32(unsafe.Sizeof(*bi))); err != nil {
 		return &os.PathError{Op: "SetFileInformationByHandle", Path: f.Name(), Err: err}
 	}
+	runtime.KeepAlive(f)
+	runtime.KeepAlive(bi)
```

Comment:
>runtime.KeepAlive(bi) [](start = 1, length = 21)

I don't think this is necessary; the cgo rules should keep bi alive across the call to setFileInformationByHandle.